modules/apps/click-to-chat/click-to-chat-web/src/main/resources/META-INF/resources/dynamic_include/zendesk.jsp

```diff
@@ -16,14 +16,44 @@
 
 <%@ include file="/dynamic_include/init.jsp" %>
 
-<script id="ze-snippet" src="https://static.zdassets.com/ekr/snippet.js?key=<%= clickToChatChatProviderAccountId %>">
-</script>
-
-<c:if test="<%= themeDisplay.isSignedIn() %>">
-	<script>
-		zE('webWidget', 'identify', {
-			email: '<%= user.getEmailAddress() %>',
-			name: '<%= user.getScreenName() %>',
-		});
-	</script>
-</c:if>
\ No newline at end of file
+<script>
+	function loadScript() {
+		function setUserInfo() {
```

Comment:
In this case, since `setUserInfo` is inside `<portlet:namespace />loadZendeskScript()` as a local function, we don't need the portlet:namespace before the function.

But, i feel this `setUserInfo` very generic, could be `setZendeskUserInfo`